lib/mayaUsd/render/vp2RenderDelegate/proxyRenderDelegate.cpp

```diff
@@ -361,11 +362,14 @@ void ProxyRenderDelegate::_InitRenderDelegate(MSubSceneContainer& container) {
 
 #if defined(WANT_UFE_BUILD)
         if (!_ufeSelectionObserver) {
+            _ufeSelectionObserver = std::make_shared<UfeSelectionObserver>(*this);
```

Comment:
Feels funny to still have it called selection observer, when it's observing both the selection and the scene...  Maybe just plain ufeObserver?

Reply:
Good point. I was thinking about the same but somehow gave up probably I thought its purpose is for selection update. But yes, I can rename it.

Reply:
Unluckily if I rename it to `_ufeObserver`, I hit a VS compiler bug that would produce an error: 

`d:\git\maya\worktrees\main\maya\src\components\usd\maya-usd\lib\mayausd\rend er\vp2renderdelegate\proxyRenderDelegate.h(201): error C2208: 'std::shared_ptr fe_v0::Observer>': no members defined using this type`

And I am not the only one hitting the issue:

https://stackoverflow.com/questions/23914815/what-is-the-error-in-my-shared-ptr-declaration

I will have to rename it to `_ufeChangeObserver`. Or if you have better suggestions?

Reply:
I am renaming it to just `_observer`.

Reply:
Works for me.